### ISSUE TITLE
10.0.19041.782 Not Supported

### ISSUE THREAD

**Issue by nturumel** [stascorp/rdpwrap#1303]
Does anyone have the ini file?

**Comment by tomkanok** [comment]
> Does anyone have the ini file?

404 :-) Waiting for ini too.

**Comment by tomkanok** [comment]
Temporary workaround:
- Uninstalling KB4598291 reverts you to supported ver. 10.0.19041.746
Tested.

**Comment by bogdik** [comment]
[10.0.19041.782]
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=88F41
LocalOnlyCode.x64=jmpshort
SingleUserPatch.x64=1
SingleUserOffset.x64=0CA4C
SingleUserCode.x64=Zero
DefPolicyPatch.x64=1
DefPolicyOffset.x64=18A15
DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
SLInitHook.x64=1
SLInitOffset.x64=1D5BC
SLInitFunc.x64=New_CSLQuery_Initialize


[10.0.19041.782-SLInit]
bInitialized.x64      =106028
bServerSku.x64        =10602C
lMaxUserSessions.x64  =106030
bAppServerAllowed.x64 =106038
bRemoteConnAllowed.x64=106040
bMultimonAllowed.x64  =106044
ulMaxDebugSessions.x64=106048
bFUSEnabled.x64       =10604C

work

**Comment by nturumel** [comment]
> [10.0.19041.782]
> LocalOnlyPatch.x64=1
> LocalOnlyOffset.x64=88F41
> LocalOnlyCode.x64=jmpshort
> SingleUserPatch.x64=1
> SingleUserOffset.x64=0CA4C
> SingleUserCode.x64=Zero
> DefPolicyPatch.x64=1
> DefPolicyOffset.x64=18A15
> DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
> SLInitHook.x64=1
> SLInitOffset.x64=1D5BC
> SLInitFunc.x64=New_CSLQuery_Initialize
> 
> [10.0.19041.782-SLInit]
> bInitialized.x64 =106028
> bServerSku.x64 =10602C
> lMaxUserSessions.x64 =106030
> bAppServerAllowed.x64 =106038
> bRemoteConnAllowed.x64=106040
> bMultimonAllowed.x64 =106044
> ulMaxDebugSessions.x64=106048
> bFUSEnabled.x64 =10604C
> 
> work

**Comment by nturumel** [comment]
Hi, Is this th eentire ini file?

**Comment by nturumel** [comment]
Hi, can you please give me the ini file that you are using?

**Comment by nturumel** [comment]
Hi, can you please give me the ini file that you are using? Right Now, it's not listening, fully supported

**Comment by AntoninoBonanno** [comment]
Hi, I tried to run update.bat, but it tells me

```
[*] Current update date: 2018.10.10
[*] Latest update date: 2018.10.10
```

and it is still "Not supported".

Where can I find the .ini file to update it manually?

-- EDIT --
I modified the file "C:\Program Files\RDP Wrapper\rdpwrap.ini" by adding bogdik's comment at the bottom,
but now the service is Stopped, and Not listening

**Comment by AntoninoBonanno** [comment]
> Hi, can you please give me the ini file that you are using? Right Now, it's not listening, fully supported

@nturumel how did you solve?

**Comment by bogdik** [comment]
See work for me
<img src="https://i114.fastpic.ru/big/2021/0205/f5/_46af9443e76ff69c1d0fbbdb1367a0f5.png?noht=1&tk=3272&md5=5Wt1QSLds3fGSKr500it-w&expires=1612530000">

**Comment by bogdik** [comment]
> how did you solve?

After change and save rdpwrap.ini try reboot

**Comment by AntoninoBonanno** [comment]
> > how did you solve?
> 
> After change and save rdpwrap.ini try reboot

I uninstalled everything. Installed from scratch, I moved all the files to "C: \ Program Files \ RDP Wrapper" updated the ini file and restarted the pc.
Now works.

Previously in the folder "C: \ Program Files \ RDP Wrapper" I had only the files "rdpwrap.ini" and "rdpwrap.dll". Now, however, I also put the release files. I believe this was the problem.

Thanks for the help
